File: src/main/java/net/glasslauncher/mods/glassguis/screen/widget/slot/BigSlot.java
package net.glasslauncher.mods.glassguis.screen.widget.slot;

import net.minecraft.inventory.Inventory;
import net.minecraft.screen.slot.Slot;

public class BigSlot extends Slot implements GlassSlot {

    public BigSlot(Inventory inventory, int index, int x, int y) {
        super(inventory, index, x, y);
    }

    @Override
    public int getWidth() {
        return 16;
    }

    @Override
    public int getHeight() {
        return 16;
    }

    @Override
    public int getBackgroundWidth() {
        return 24;
    }

    @Override
    public int getBackgroundHeight() {
        return 24;
    }
}

File: src/main/kotlin/net/glasslauncher/mods/glassguis/BigSlot.kt
package net.glasslauncher.mods.glassguis

import net.minecraft.inventory.Inventory
import net.minecraft.screen.slot.Slot

open class BigSlot(inventory: Inventory, index: Int, x: Int, y: Int) : Slot(inventory, index, x, y), CustomSizeSlot {
    override fun getWidth(): Int {
        return 16
    }

    override fun getHeight(): Int {
        return 16
    }

    override fun getBackgroundWidth(): Int {
        return 24
    }

    override fun getBackgroundHeight(): Int {
        return 24
    }
}